src/main/webapp/app/service/firebase/firebase-gene-review-service.ts

```diff
@@ -168,9 +169,14 @@ export class FirebaseGeneReviewService {
         FIREBASE_LIST_PATH_TYPE.MUTATION_LIST,
       ]) {
         for (const [firebasePath, deleteIndices] of Object.entries(itemsToDelete[pathType])) {
+          hasDeletion = true;
           await this.firebaseRepository.deleteFromArray(firebasePath, deleteIndices);
         }
       }
+      // If user accepts a deletion individually, we need to refresh the ReviewPage with the latest data to make sure the indices are up to date.
+      if (reviewLevels.length === 1 && hasDeletion) {
```

Comment:
Why do we only refresh if the length is 1? If a review level corresponds to some change that needs to be reviewed, wouldn't we want to refresh if length > 0?

Reply:
The only time reviewLevels length is > 1 is when we use the accept all feature. In that case a refresh is not needed because everything should be accepted all together.